car_rental/demo/website_theme_apply.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo>
```

Comment:
Do you need this file? There is no theme to apply...

Reply:
Yes we need this file